docs/README.md

```diff
@@ -0,0 +1,100 @@
+# ethereumjs-wallet
+
+## Index
+
+### Classes
+
+- [EthereumHDKey](classes/ethereumhdkey.md)
+- [Wallet](classes/wallet.md)
+- [WalletSubprovider](classes/walletsubprovider.md)
+
+### Interfaces
+
+- [EtherWalletOptions](interfaces/etherwalletoptions.md)
+- [EvpKdfOpts](interfaces/evpkdfopts.md)
+
+### Functions
+
+- [fromEtherCamp](#fromethercamp)
+- [fromEtherWallet](#frometherwallet)
+- [fromKryptoKit](#fromkryptokit)
+- [fromQuorumWallet](#fromquorumwallet)
+
+---
+
+## Functions
+
+<a id="fromethercamp"></a>
+
+### fromEtherCamp
+
+▸ **fromEtherCamp**(passphrase: _`string`_): [Wallet](classes/wallet.md)
+
+_Defined in [thirdparty.ts:165](https://github.com/alcuadrado/ethereumjs-wallet/blob/8b5f5a9/src/thirdparty.ts#L165)_
```

Comment:
GitHub URLs are referencing your repository. Can you regenerate the docs after the requested changes applied and resubmit the PR towards the main repo? Have granted you sufficient permissions to do so.

Reply:
Done